### DISCUSSION TITLE
Question: is there an easy way to grab the data into a csv

### DISCUSSION THREAD

**Discussion by biohazd** [HandyHat/ha-hildebrandglow-dcc#283] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
is there an easy way to grab the data into a csv file ? I want to play with it in Excel

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by ColinRobbins** [comment]
The data is all in the SQL database, you could export it from there

**Comment by HandyHat** [comment, marked as answer]
Hey @biohazd,
This is a bit out of scope for this integration but you could use pyglowmarkt to output readings into csv
https://github.com/cybermaggedon/pyglowmarkt#glowmarkt-csv
